app/src/main/cpp/src/appMain/sample_policy_manager.py

```diff
@@ -70,8 +76,12 @@ def pack(data, encryption, add_http_header):
     return file_path
 
 
-def unpack(data, encryption):
+def unpack(data, encryption, android):
     file_path = data['fileName']
+
+    if android:
+        path = "/tmp/sdl_snapshot.json"
+
```

Comment:
@VadymLuchko same as above

Reply:
@AKalinich-Luxoft fixed in e0fd5100b84e13d8c930043276b9b3f9d0ee5c3f